todo-app/src/TodoList.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {inject, observer} from 'mobx-react';
+import Todo from './Todo';
+
+@inject('todos')@observer
+export default class TodoList extends React.Component {
+
+  state = {
+    editingTodoID: null,
```

Comment:
I think it would be sensible to move this `editingTodoID` into your MobX store. You currently have state in at least four places:

1. MobX store
2. Here in `TodoList`
3. In `TodoForm` for creating new todos
4. In `Todo` for modifying existing todos

Let's see if you can lift some of this state out of React components and into your MobX store.